_pages/en/it-picture-long-term.md

```diff
@@ -87,35 +87,59 @@ Ecosystem of services are created by leveraging private sector expertise where o
 
 #### Modern Open Organization
 
-IITB is a modern open organization with a very open culture.
+##### Enterprise Architecture
+
+Enterprise Architecture (EA) committees and review boards at IITB-ESDC have aligned with TBS mandatory procedures for EA (users, open standards, open source software and cloud first).
 Wherever possible code and projects are developed in the open and use of open source software is the norm.
-IITB is the digital, IM and IT advisor and enabler for ESDC and part of major business decisions.
-IITB is responsible for providing the automated services required to run a secure and agile business.
-This includes, automated software onboarding approval, key data exchange systems with which business may interact with, and an environment by which business may securely deploy their own solutions.
+EA teams enforce a loosely coupled architecture that empowers development and operations teams to meet business needs.
```

Comment:
Instead of  'enforce' I would say provide. 

Example: EA is responsible for managing our artifact repository. If you use all libraries from said repository then you are compliant. If not, you are not. 

Each group has the option to use or not use the libraries. EA is measured against the number of people who opt into compliance.